README.md

```diff
@@ -123,80 +129,78 @@ Check if a given thing is Runnable
 | --- | --- |
 | thing | <code>\*</code> | 
 
+<a name="LinkerIterator"></a>
+
+## LinkerIterator
+Class LinkerIterator returns the next value for Iterable classes.
+
+**Kind**: global class  
+<a name="ArrayIterator"></a>
+
+## ArrayIterator
+Class ArrayIterator returns the next value for Iterable classes.
+
+**Kind**: global class  
```

Comment:
These descriptions appear a second time and they are the same which needs to be improved.